### DISCUSSION TITLE
(wrong) Timezone is added to the time in the new post,

### DISCUSSION THREAD

**Discussion by sardarms** [sveltia/sveltia-cms#138] [General]
Hi, I just realised that when I create any new post, in the date, a timezone is added. Not sure why. And since the timezone was ahead of me, none of my post were shown my the site and I spend almost 2 days to figure it out. 

I finally fixed the issue with this code: 

`      - { label: 'Publish Date', name: 'date', widget: 'datetime', dateFormat: 'YYYY-MM-DD', format: 'YYYY-MM-DD', required: false }`

It seems like this issue were discussed in the past: 

https://github.com/the-new-developers/tnd-website/issues/32

https://stackoverflow.com/questions/55550029/date-widget-input-appears-to-save-inconsistent-to-display 

This post is a reminder for me, if future me face the problem again.

**Comment by kyoshino** [comment]
@sardarms Thanks for the heads up! Just quickly looked at the Sveltia code and probably found the cause. Will test and release a fix.

**Reply by kyoshino** [reply]
Shipped [v0.25.1](https://github.com/sveltia/sveltia-cms/releases/tag/v0.25.1) with a fix! I’ll write automated tests soon to make sure every combination of the widget options is correct.

**Comment by sardarms** [comment]
@kyoshino thanks. I spent two days pulling my hair trying to find the issue. 😂

**Reply by kyoshino** [reply]
Sorry for the trouble 🙏🏼 If you find any other problems in the future, please let me know.

**Comment by sardarms** [comment]
@kyoshino  for what it's worth, the same problem exists in Decap CMS. I have tried both.

**Reply by kyoshino** [reply]
Sveltia and Decap CMS are completely different products with no shared code, but still we might have the same problem. I’m happy to fix any bugs to make Sveltia a much better alternative 😸
